hice_backend/settings.py

```diff
@@ -13,10 +14,11 @@
 
 SECRET_KEY = os.getenv("DJANGO_SECRET_KEY")
 
-if os.getenv("DJANGO_ENV") == "development":
-    DEBUG = True
-else:
-    DEBUG = False
+ENVIRONMENT = os.getenv("DJANGO_ENV")
+if not ENVIRONMENT:
+    ENVIRONMENT = "production"
```

Comment:
I think it would be better to default to development here; I just added the key-value pair `DJANGO_ENV` and `production` to our remote back-end env.

Reply:
Easy enough to add it.